backend.native/tests/interop/objc/allocException.kt

```diff
@@ -18,7 +18,13 @@ fun exc_handler(x: Any?) : Unit {
 }
 
 fun main() {
+    // This does not work anymore, as NSException propagated as ForeignException
```

Comment:
I guess the new test version works with old behaviour too, but can be reverted now.
Please don't forget to revert if you find it necessary.

Reply:
I'd rather keep it as is. This test is intended to validate that snippets runs "as expected", as opposite to segmentation fault (as it was before), or another OS level fault. Both variants are expected now with different cinterop options, so both cases are accepted in this test. Let's have another test to validate different behavior with different options.